veggieseasons/lib/screens/list.dart

```diff
@@ -4,19 +4,65 @@
 
 import 'package:flutter/cupertino.dart';
 import 'package:flutter/widgets.dart';
+import 'package:intl/intl.dart';
+import 'package:scoped_model/scoped_model.dart';
+import 'package:veggieseasons/data/model.dart';
+import 'package:veggieseasons/data/veggie.dart';
 import 'package:veggieseasons/styles.dart';
+import 'package:veggieseasons/widgets/veggie_headline.dart';
 
 class ListScreen extends StatelessWidget {
+  List<Widget> _generateVeggieRows(List<Veggie> veggies) {
+    final cards = new List<Widget>();
+
+    for (Veggie veggie in veggies) {
+      cards.add(Padding(
+        padding: EdgeInsets.only(left: 16.0, right: 16.0, bottom: 24.0),
+        child: VeggieHeadline(veggie),
+      ));
+    }
+
+    return cards;
+  }
+
   @override
   Widget build(BuildContext context) {
-    return CupertinoPageScaffold(
-      navigationBar: CupertinoNavigationBar(
-        middle: Text('List'),
+    String dateString = DateFormat.yMMMMd("en_US").format(DateTime.now());
+    final model = ScopedModel.of<AppState>(context, rebuildOnChange: true);
```

Comment:
I _think_ `rebuildOnChange` is `true` by default, in which case you can drop this.

Reply:
I think it may have been in the past, but the default is `false` now:

https://github.com/brianegan/scoped_model/blob/master/lib/scoped_model.dart#L168